src/main/java/uk/nhs/digital/nhsconnect/nhais/utils/MissingOrEmptyElementUtils.java

```diff
@@ -0,0 +1,25 @@
+package uk.nhs.digital.nhsconnect.nhais.utils;
+
+import java.util.List;
+
+import uk.nhs.digital.nhsconnect.nhais.exceptions.FhirValidationException;
+
+public class MissingOrEmptyElementUtils {
+
+    public static void exceptionIfMissingOrEmpty(String path, Object value) throws FhirValidationException {
```

Comment:
To many `MissingOrEmpty` - Maybe name class `FhirElementUtils`

Reply:
Also this method could be private and the other ones (like `checkGpPresence()`) could be moved into this class. This would greatly simplify other classes where you wouldn't have to manually add `path` param

Reply:
I just copied it from our code :) this method was in three places

Reply:
I think then renaming class is a good idea

Reply:
ok, I will move this checkGpPresence